images/win/scripts/Tests/WinAppDriver.Tests.ps1

```diff
@@ -0,0 +1,18 @@
+Describe "WinAppDriver" {
+  It "WinAppDriver directory exists" {
+    $wad = "Windows Application Driver"
+    if (${env:ProgramFiles(x86)})
```

Comment:
useless check all systems have x86

Reply:
The code inherited from the Validation script. I guess this 2016/2019 branching. To remove it there must be additional investigation

Reply:
Yes, please do it the additional investigation and remove the redundancy code.